### PR TITLE
Fixes in release/Jenkinsfile

### PR DESCRIPTION
**Pull request by vivek**
# Description

This was used to release 1.1.0-beta-8 version.

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
:bee: LGTM

File: release/Jenkinsfile [hunk 1]
```diff
@@ -12,63 +15,70 @@ node {
...
+                }
+
+                //Sanity build
+                stage('Build') {
+                    sh 'echo Java version: `java -version`'
+                    sh 'echo Maven version: `mvn --version`'
+                    sh 'echo Git version: `git --version`'
+
+                    // clean build
+                    sh 'mvn -B -T 1C clean install -Djacoco.skip=true -DskipTests -Dfindbugs.skip=true'
```
Comment (michaelneale) [inline release/Jenkinsfile]: oops I mean here :bug: - this -T 1C was causing the failure due to concurrent downloading.

Comment (michaelneale) [inline release/Jenkinsfile]: so need to drop that

Comment (vivek) [inline release/Jenkinsfile]: @michaelneale done. Also removed `mvn clean` hack which was apparently due to maven frontend end concurrency issue. PTAL.

**Comment by michaelneale** [comment]
:bee:

**Pull request merged**